deployment/kustomizations/base/pvc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
```

Comment:
I think we should add a `PersistentVolume` as an example as well ?

Reply:
No, PVC will create PV automatically.

Reply:
I'm curious, does that work on all storageclass ? or just some ?

Reply:
this is how it works by default in k8s

Reply:
I think it's only supported by k8s environments that have [Dynamic Volume Provisioning](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/), but sure, since I also saw most cloud providers and also minikube supporting this, I think it's fine.